tools/vmlist.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
+
+import ConfigParser
+import docopt
+import multiprocessing
+import novaclient.client
+import os
+import subprocess
+import sys
+import tempfile
+import textwrap
+
+CACHEFILE = "~/.vmlist.cache"
+CONFFILE = "~/.vmlist.conf"
+
+# mira004.front.sepia.ceph.com is dead
+
+VMMACHINES = textwrap.dedent('''\
```

Comment:
How about VM_HOSTS? Also, how about letting it be overridden by setting an env variable of the same name?